src/ckb/network.rs

```diff
@@ -424,10 +419,6 @@ where
                 }
             }
             NetworkActorEvent::ChannelReady(channel_id, peer_id) => {
-                info!(
```

Comment:
I added these messages as `info!` because I thought it would be both useful to developer and users. Users sometimes want to learn that some channel is actually closed on their command.

Reply:
reverted, please review again, thanks.